### PR TITLE
line added on feature branch

### PR DESCRIPTION
**Pull request by vivekboora**
check out my repo [here](https://github.com/vivekboora/GitPractice.git)

### CODE REVIEW

**Comment by outttofthebox** [comment]
this is a nice addition @vivekboora  but could you add one more line saying 'Hello world how are you' in #1

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # GitPractice
 adding a new line
```
Comment (outttofthebox) [inline README.md:2]: change it to brand new line

**Comment by vivekboora** [comment]
This is where you need to make modifications https://github.com/outttofthebox/GitPractice/blob/4cdf0baf9ec4dedfc13ec5b1b561f9b1925625cc/README.md?plain=1#L2-L3

**Comment by outttofthebox** [comment]
Here you need to make modifications https://github.com/outttofthebox/GitPractice/blob/4cdf0baf9ec4dedfc13ec5b1b561f9b1925625cc/README.md?plain=1#L2-L3

**Pull request merged**